python/hail/table.py

```diff
@@ -1825,7 +1827,7 @@ def flatten(self):
                 }]
             }
 
-        and a single key field ``a``.  The result of flatten is
+        and a single key ``a``.  The result of flatten is
```

Comment:
remove "a single" since multiple keys are not possible

Reply:
Good catch.